tests/data_access/test_from_lims.py

```diff
@@ -1,11 +1,272 @@
+    assert meso_table["specimen_name"][0] == 'Sst-IRES-Cre;Ai148-457841'
+
+    assert meso_table["session_type"][0]       == 'OPHYS_6_images_B'
+    assert meso_table["targeted_structure"][0] == 'VISp'
+    assert meso_table["depth"][0]              == 225
+    assert meso_table["equipment_name"][0]     == 'MESO.1'
+    assert meso_table["project"][0]            == 'VisualBehaviorMultiscope'
+
+
+@pytest.mark.onprem
+def test_get_ophys_experiment_ids_for_ophys_session_id():
+    meso_table = from_lims.get_ophys_experiment_ids_for_ophys_session_id(OPHYS_SESSION_ID_MESO)
+    assert meso_table["ophys_experiment_id"][0] == 960410023
+    assert meso_table["ophys_experiment_id"][1] == 960410026
+    assert meso_table["ophys_experiment_id"][2] == 960410028
+    assert meso_table["ophys_experiment_id"][3] == 960410032
+    assert meso_table["ophys_experiment_id"][4] == 960410035
+    assert meso_table["ophys_experiment_id"][5] == 960410038
+    assert meso_table["ophys_experiment_id"][6] == 960410040
+    assert meso_table["ophys_experiment_id"][7] == 960410042
+
+
+@pytest.mark.onprem
+def test_get_behavior_session_id_for_ophys_session_id():
+    assert from_lims.get_behavior_session_id_for_ophys_session_id(OPHYS_SESSION_ID_MESO) == 959681045
+
+
+@pytest.mark.onprem
+def test_get_ophys_container_ids_for_ophys_session_id():
+    sci_table = from_lims.get_ophys_container_ids_for_ophys_session_id(OPHYS_SESSION_ID_SCI)
+    assert sci_table["ophys_container_id"][0] == CONTAINER_ID_SCI
+
+    meso_table = sci_table = from_lims.get_ophys_container_ids_for_ophys_session_id(OPHYS_SESSION_ID_MESO)
+    assert meso_table["ophys_container_id"][0] == 1018028342
+    assert meso_table["ophys_container_id"][1] == 1018028345
+    assert meso_table["ophys_container_id"][2] == 1018028339
+    assert meso_table["ophys_container_id"][3] == 1018028348
+    assert meso_table["ophys_container_id"][4] == 1018028354
+    assert meso_table["ophys_container_id"][5] == 1018028357
+    assert meso_table["ophys_container_id"][6] == 1018028351
+    assert meso_table["ophys_container_id"][7] == 1018028360
+
+
+@pytest.mark.onprem
+def test_get_supercontainer_id_for_ophys_session_id():
+    assert from_lims.get_supercontainer_id_for_ophys_session_id(OPHYS_SESSION_ID_SCI)  == None
+    assert from_lims.get_supercontainer_id_for_ophys_session_id(OPHYS_SESSION_ID_MESO) == 1027966320
+
+
+@pytest.mark.onprem
+def test_get_all_ids_for_ophys_session_id():
+    sci_table = from_lims.get_all_ids_for_ophys_session_id(OPHYS_SESSION_ID_SCI)
+    assert sci_table["ophys_experiment_id"][0] == OPHYS_EXPERIMENT_ID_SCI
+    assert sci_table["ophys_session_id"][0]    == OPHYS_SESSION_ID_SCI
+    assert sci_table["behavior_session_id"][0] == BEHAVIOR_SESSION_ID_SCI
+    assert sci_table["ophys_container_id"][0]  == CONTAINER_ID_SCI
+    assert sci_table["supercontainer_id"][0]   == None
+
+    meso_table = from_lims.get_all_ids_for_ophys_session_id(OPHYS_SESSION_ID_MESO)
```

Comment:
For the meso table, there multiple rows because there is more than one experiment for each session. I wanted to be sure that it's not jus the first experiment that is accurate so I tested different points in the table. 
![image](https://user-images.githubusercontent.com/37673624/121093681-38698180-c7a2-11eb-94b5-17061940b27d.png)